docs/clients/coil_temperature_estimator_full.rst

```diff
@@ -0,0 +1,158 @@
++--------+------------------------------+----------------+-----------+--------------+-------------------------------------------------------------------------------------------------------------------------+
+| 3      | h_coil_amb_free_conv         | get, set, save | float     |              | Free convection heat transfer coefficient used when the motor is not spinning                                           |
++--------+------------------------------+----------------+-----------+--------------+-------------------------------------------------------------------------------------------------------------------------+
+| 4      | h_coil_stator_cond           | get, set, save | float     |              | Conduction heat transfer coefficient between the coils and the stator aluminum                                          |
++--------+------------------------------+----------------+-----------+--------------+-------------------------------------------------------------------------------------------------------------------------+
+| 5      | h_coil_amb_forced_conv       | get            | float     |              | The present calculated force heat transfer convection coefficient.                                                      |
++--------+------------------------------+----------------+-----------+--------------+-------------------------------------------------------------------------------------------------------------------------+
+| 6      | c_coil                       | get, set, save | float     |              | Thermal heat capacitance/mass of the coils                                                                              |
++--------+------------------------------+----------------+-----------+--------------+-------------------------------------------------------------------------------------------------------------------------+
+| 7      | h_coil_amb_forced_conv_coeff | get, set, save | float     |              | Forced convection coefficient calculation coefficient, h_forced_conv = h_conv_coeff * sqrt(speed)                       |
++--------+------------------------------+----------------+-----------+--------------+-------------------------------------------------------------------------------------------------------------------------+
+| 8      | otw                          | get, set, save | float     | degC         | Over temperature warning, Derating of the motor begins at this temperature                                              |
++--------+------------------------------+----------------+-----------+--------------+-------------------------------------------------------------------------------------------------------------------------+
+| 9      | otlo                         | get, set, save | float     | degC         | Over temperature lock out, Derating of the motor end at this temperature, where the motor is fully disabled             |
```

Comment:
ends

Reply:
And Derating should be derating.  No caps needed here.